README.md

```diff
@@ -1,2 +1,26 @@
-# Hello-Microverse
-This is a hello microverse project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-microverse
+
+> Hello microverse description.
+
+## Built With
+
+html and css
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+👤 **Author1**
+
+- GitHub: https://github.com/Danelif
+
```

Comment:
kindly add your contacts for example your LinkedIn and your Twitter. know that the contact section is very important because it easily allows recruiters or people interested in your project to contact you without difficulty.